public-ui/src/app/data-browser/views/pm/pm-react.component.tsx

```diff
@@ -0,0 +1,488 @@
+                     const btnClass = selectedConcept === concept ? 'btn btn-link concept-button active' : 'btn-link btn concept-button';
+                     return <button className={btnClass} key={index} onClick={() => this.showMeasurement(selectedGroup, concept)}
+                       style={styles.btnLink}>{concept.conceptName}</button>;
+                     })
+                   }
+                   </div>
+                   : null
+                }
+                </div>
+                {  unitNames && unitNames.length > 1 ?
+                     <div className='group-unit'>
+                     {  unitNames.map((unit, index) => {
+                          const btnClass = selectedConceptUnit === unit ? 'btn btn-link unit-button active' : 'btn btn-link unit-button';
+                          return <button className={btnClass} key={index} onClick={() => this.setConceptUnit(unit)}
+                            style={styles.btnLink}>{unit}</button>;
+                          })
+                     }
+                     </div> : null
+                     }
+                     {  selectedConcept && (selectedConcept.conceptId === '903111' || selectedConcept.conceptId === '903120') ?
+                        selectedConcept.analyses.countAnalysis.results[0].countValue > 20 ?
+                        <div className='participant-count'>
+                          Total Participant count: {selectedConcept.analyses.countAnalysis.results[0].countValue}
+                        </div> :
+                        <div className='participant-count'>
+                          Total Participant count: &le; {selectedConcept.analyses.countAnalysis.results[0].countValue}
+                        </div> : null
+                     }
+                     <div className='chart-layout' style={styles.chartLayout}>
+                       {  selectedConcept && selectedConcept.analyses && selectedConcept.analyses.measurementGenderCountAnalysis ?
+                          selectedConcept.conceptId !== '903111' && selectedConcept.conceptId !== '903120' && selectedConceptUnit ?
+                          <React.Fragment>
+                          {  this.getCountAnalysis(selectedConceptUnit).map((gender, index) => {
+                               const chartKey = gender.stratum3 + '-' + index;
+                               return <div className='bs-chart-item' key={chartKey}>
+                                 <ValueReactChartComponent conceptId={selectedConcept.conceptId}
+                                   valueAnalysis={this.getValueAnalysis()}
+                                   domainCountAnalysis={domainCountAnalysis}
+                                   genderId={gender.stratum3}
+                                   chartTitle={gender.analysisStratumName + ' - ' + (gender.countValue <= 20 ? '&le; ' : '') +
+                                   gender.countValue.toLocaleString()} key={chartKey}></ValueReactChartComponent></div>;
```

Comment:
Use the self-closing syntax:
```suggestion
                                   gender.countValue.toLocaleString()} key={chartKey}/>
                                 </div>;
```

Reply:
Thanks. changed this.